notifirehq/notifire
# feat(novui,web): V2 flow list

### What changed? Why was the change needed?


- Create table in `novui` with newest version of react-table
  - Add stories
- Add design tokens from Figma
  - _Note: I'm not thrilled with how some of them are done, but I've kept them as-is to match Figma for the time being_
- Setup test example of new Workflows page in `web`

### Validation

1. Run `pnpm storybook` in `novui` and look for Table
2. Visit https://deploy-preview-5682--dev-web-novu.netlify.app/studio/flows to see in web

### Screenshots


![Screenshot 2024-06-06 at 11 22 56 AM](https://github.com/novuhq/novu/assets/47441786/fb32bff9-fdcc-4f62-8be5-686aa97e0073)
![Screenshot 2024-06-06 at 11 25 47 AM](https://github.com/novuhq/novu/assets/47441786/8fcc1b7f-aad8-4089-870f-1303cb0588d0)
